iib/web/api_v1.py

```diff
@@ -542,18 +559,30 @@ def add_rm_batch():
             safe_args[safe_args.index(build_request['overwrite_from_index_token'])] = '*****'
 
         error_callback = failed_request_callback.s(request.id)
-        if isinstance(request, RequestAdd):
-            handle_add_request.apply_async(
-                args=args, link_error=error_callback, argsrepr=repr(safe_args), queue=celery_queue,
-            )
-        else:
-            handle_rm_request.apply_async(
-                args=args, link_error=error_callback, argsrepr=repr(safe_args), queue=celery_queue,
-            )
+        try:
+            if isinstance(request, RequestAdd):
+                handle_add_request.apply_async(
+                    args=args,
+                    link_error=error_callback,
+                    argsrepr=repr(safe_args),
+                    queue=celery_queue,
+                )
+            else:
+                handle_rm_request.apply_async(
+                    args=args,
+                    link_error=error_callback,
+                    argsrepr=repr(safe_args),
+                    queue=celery_queue,
+                )
+        except OperationalError:
```

Comment:
Can you be consistent and either always use `OperationalError` or `kombu.exceptions.OperationalError`?

Reply:
Thanks, changed to kombu.exceptions.OperationalError